src/test/java/CartTests.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import shop.Cart;
+import shop.RealItem;
+
+public class CartTests {
+
+    Cart cart;
+    RealItem car;
+
+    @Before
+    public void beforeTest() {
+        cart = new Cart("andrew-cart");
+        car = new RealItem();
+    }
+
+    @Test
+    public void testPriceCalculationWhenAddingItem() {
+        car.setName("Audi");
+        car.setPrice(32026.9);
+        car.setWeight(1560);
```

Comment:
you can move this common logic into  @BeforeEach method

Reply:
Fixed